tests/modules/programs/bash/bash-history-control-with-file.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.bash = {
+      enable = true;
+      historyControl = [ "erasedups" ];
+      historyFile = "/home/hm-user/foo/bash/history";
+    };
+
+    nmt.script = ''
+        assertFileExists home-files/.bashrc
```

A bit too much indentation:
```suggestion
     assertFileExists home-files/.bashrc
```

---

fixed, thank you